config/manager/manager.yaml

```diff
@@ -65,10 +65,10 @@ spec:
         # More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
         resources:
           limits:
-            cpu: 500m
-            memory: 128Mi
+            cpu: 1000m
+            memory: 512Mi
           requests:
-            cpu: 10m
-            memory: 64Mi
+            cpu: 1000m
+            memory: 512Mi
```

Comment:
I think we should try increasing only the `limits`, or at least increase the `requests` only a little bit.

Reply:
having them equal will place the pod in the best QOS

Reply:
That's for sure :)
Anyhow, alright. I'm ok with that.